tests/istio.io/tasks/security/authz-http/output/bookinfo-gateway.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: bookinfo-gateway
+spec:
+  selector:
+    istio: ingressgateway # use istio default controller
+  servers:
+  - port:
+      number: 80
+      name: http
+      protocol: HTTP
+    hosts:
+    - "*"
+---
+apiVersion: networking.istio.io/v1alpha3
```

Comment:
is `output` supposed to be checked in?

Reply:
I was talking with @geeknoid about this earlier. I think what will happen is that this will be cached in the test results and then referenced on Istio.io from there. So the output shouldn't be here, but serves as a good example of what it looks like. Removing it so it doesn't accidentally get merged. Thanks.